lib/plugins/inspec-license/lib/inspec-license/cli.rb

```diff
@@ -7,6 +7,9 @@ class CLI < Inspec.plugin(2, :cli_command)
     desc "list", "List licenses (not applicable to local licensing service)"
     def list
       ChefLicensing.list_license_keys_info
+    rescue ChefLicensing::Error => e
+      Inspec::Log.error e.message
+      Inspec::UI::EXIT_LICENSE_NOT_SET
```

Comment:
This is wrong. You should call ui.exit(), not just return a code.

Reply:
Good catch @clintoncwolfe. I did not read it thoroughly.